perf_benchmark/benchmark_main.cc

```diff
@@ -172,11 +252,19 @@ void SinglePayloadFromJson(::benchmark::State& state, uint64_t payload_length,
                            bool streaming, uint64_t stream_size) {
   std::string json_msg = absl::StrFormat(
       R"({"payload" : "%s"})", GetRandomBytesString(payload_length, true));
-  absl::Status status = BenchmarkJsonTranslation(
-      state, kBytesPayloadMessageType, json_msg, streaming, stream_size, 1);
-  if (!status.ok()) {
-    state.SkipWithError(status.ToString().c_str());
-  }
+  SkipWithErrorIfNotOk(
```

Comment:
Still prefer explicitly calling `BenchmarkJsonTranslation`,  it is easier to read.  Maybe?

auto status = BenchmarkJsonTranslation();
SkipWithErrorIfNotOk(state, status);